### PR TITLE
Changes done for "The Cents Club" collection.

### PR DESCRIPTION
**Pull request by xkiranj**
ENS-COLLECTIONS:
- New file "the-cents-club.csv" is added to the collections, with all the possible values for "Cents" ENS domains.

- New logo file "the-cents-club.png" is added.

- Following metadata is added to the "ens-collections.json" file.
		{
			"name": "The Cents Club",
			"slug": "the-cents-club",
			"description": "Ens domains for 1cent to 100cents",
			"twitter": "TheCentsClub",
			"chat": "https://discord.gg/YbNGdmCNwT",
			"website": "",
			"logo": "the-cents-club.png",
			"csv": ["the-cents-club.csv"]
		}

### CODE REVIEW

**Comment by xkiranj** [comment]
Did the changes to submit the collection for "The Cents Club".
Thank You!

File: collections/the-cents-club.csv [hunk 1]
```diff
@@ -0,0 +1,100 @@
+1cent
```
Comment (AudriusHelloWorld) [inline collections/the-cents-club.csv:1]: bad file.
example
1cent,5243523452345234525234235435

read readme.md file

Comment (xkiranj) [reply collections/the-cents-club.csv:1]: Hi,

Sorry, I could not understand what the issue was.
Can you please help me here so I can fix the issue and resubmit.

Thanks!!

Comment (xkiranj) [reply collections/the-cents-club.csv:1]: I followed these instructions:
image

**Comment by xkiranj** [comment]
Updated the .cvs file for "the-cents-club" with the TokenIDs.
Thanks!!

**Pull request merged**